design-documents/graph-ql/coverage/catalog/configurable-options-selection.graphqls

```diff
@@ -0,0 +1,20 @@
+type ConfigurableProduct {
+    configurable_options_selection_metadata(selectedConfigurableOptionValues: [ID!]): ConfigurableOptionsSelectionMetadata @doc(description: "Metadata for the specified configurable options selection")
```

Comment:
[Similar Comment](https://github.com/magento/architecture/pull/394/files#r452970896)

Maybe we can drop the `configurable` prefix here? If you're working with a `ConfigurableProduct`, you typically are fetching it from a field using `ProductInterface`, so a client query already has to include `...on ConfigurableProduct`.

Based on that, I think we can drop the `configurable_` prefix, because it's assumed that a configurable product has configurable options.

Thoughts?

Reply:
Se response in another related comment https://github.com/magento/architecture/pull/394#discussion_r453756442